spec/controllers/v1.0/actions_controller_spec.rb

```diff
@@ -66,8 +66,8 @@
     end
 
     context 'when request is actionable' do
-      let!(:stage1) { create(:stage, :id => "1", :state => Stage::NOTIFIED_STATE, :request => req, :tenant_id => tenant.id) }
-      let!(:stage2) { create(:stage, :id => "2", :state => Stage::PENDING_STATE, :request => req, :tenant_id => tenant.id) }
```

Comment:
Were these changes needed as part of the basic auth change?

Reply:
no. it failed the build. was introduced by a previous PR.